oap-server/banyandb-java-client/src/main/proto/banyandb/v1/banyandb.proto

```diff
@@ -32,44 +32,18 @@ message Metadata {
   string name = 2;
 }
 
-// IntPair in a typed pair with an int64 as values
-message IntPair {
-  string key = 1;
-  int64 value = 2;
-  bool isNull = 3;
-}
-
-// StrPair in a typed pair with a string as values
-message StrPair {
-  string key = 1;
-  string value = 2;
-  bool isNull = 3;
-}
-
-// IntPair in a typed pair with an array of int64 as values
-message IntArrayPair {
-  string key = 1;
-  repeated int64 value = 2;
-  bool isNull = 3;
-}
-
-// StrPair in a typed pair with an array of string as values
-message StrArrayPair {
-  string key = 1;
-  repeated string value = 2;
-  bool isNull = 3;
-}
-
 // Pair is the building block of a record which is equivalent to a key-value pair.
 // In the context of Trace, it could be metadata of a trace such as service_name, service_instance, etc.
 // Besides, other fields/tags are organized in key-value pair in the underlying storage layer.
 // One should notice that the values can be a multi-value.
 message TypedPair {
+  string key = 1;
+  bool  is_null =  2;
```

Comment:
You moved is_null out of time, how does banyandb server reponse when int type is null? What will HasInt include?

Reply:
has introduced a new null type which contains an underlying field type